### PR TITLE
feat: Update org units core #6775 and basic views for Units #6544

### LINKED ISSUE #6775: Update org units core

### PR DESCRIPTION
**Pull request by Pucek9**
Closes #6775 #6544

### CODE REVIEW

File: projects/core/src/occ/adapters/organization/converters/occ-org-unit-node-list-normalizer.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { Occ } from '../../../occ-models/occ.models';
+import { Converter } from '../../../../util/converter.service';
+import { B2BUnitNode } from '../../../../model/org-unit.model';
+
+@Injectable()
+export class OccOrgUnitNodeListNormalizer
+  implements Converter<Occ.B2BUnitNodeList, B2BUnitNode[]> {
+  constructor() {}
```
Comment (Zeyber) [inline projects/core/src/occ/adapters/organization/converters/occ-org-unit-node-list-normalizer.ts]: Constructor needed?

File: projects/core/src/occ/adapters/organization/converters/occ-org-unit-node-normalizer.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { Occ } from '../../../occ-models/occ.models';
+import { Converter } from '../../../../util/converter.service';
+import { B2BUnitNode } from '../../../../model/org-unit.model';
+
+@Injectable()
+export class OccOrgUnitNodeNormalizer
+  implements Converter<Occ.B2BUnitNode, B2BUnitNode> {
+  constructor() {}
```
Comment (Zeyber) [inline projects/core/src/occ/adapters/organization/converters/occ-org-unit-node-normalizer.ts]: Constructor needed?

File: projects/core/src/organization/facade/org-unit.service.ts [hunk 3]
```diff
@@ -20,29 +22,49 @@ export class OrgUnitService {
     protected authService: AuthService
   ) {}
 
-  private loadOrgUnit(orgUnitId: string) {
+  loadOrgUnit(orgUnitId: string) {
```
Comment (Zeyber) [inline projects/core/src/organization/facade/org-unit.service.ts]: Should have a return type (same with other methods here)

File: projects/core/src/organization/store/reducers/org-unit.reducer.ts [hunk 4]
```diff
@@ -19,8 +19,6 @@ export function orgUnitListReducer(
   action: LoaderAction
 ): any {
   switch (action.type) {
```
Comment (Zeyber) [inline projects/core/src/organization/store/reducers/org-unit.reducer.ts:20]: Why are empty switch statements here in this file?

File: projects/core/src/organization/store/selectors/org-unit.selector.spec.ts [hunk 5]
```diff
@@ -1,7 +1,11 @@
 import { Type } from '@angular/core';
 import { TestBed } from '@angular/core/testing';
 import { select, Store, StoreModule } from '@ngrx/store';
-import { B2BUnitNode } from '../../../model/org-unit.model';
+import {
+  // B2BApprovalProcess,
```
Comment (Zeyber) [inline projects/core/src/organization/store/selectors/org-unit.selector.spec.ts]: Commented out

File: projects/storefrontlib/src/cms-components/organization/units/unit-details/unit-details.component.html [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+            "
+            class="btn btn-secondary btn-block"
+          >
+            {{ 'orgUnit.back' | cxTranslate }}
+          </a>
+        </div>
+      </div>
+    </div>
+  </div>
+  <!--  <div class="col-md-12 col-lg-6">-->
```
Comment (Zeyber) [inline projects/storefrontlib/src/cms-components/organization/units/unit-details/unit-details.component.html]: Should leave a comment for the purpose of commented out code or remove it

**Review by Zeyber — CHANGES_REQUESTED** [review]

File: projects/core/src/organization/store/reducers/org-unit.reducer.ts (hunk 4 above)
Comment (Pucek9) [reply projects/core/src/organization/store/reducers/org-unit.reducer.ts:20]: most of that reducers are not used, but could be if we will not get updated api, lets do cleanup at the end of epic

**Pull request merged**